custom/code_types.inc.php

```diff
@@ -144,6 +144,9 @@ function define_external_table(&$results, $index, $table_name,$col_code, $col_de
 
 // For generic concepts, use the fully specified description (DescriptionType=3) so we can tell the difference between them.
 define_external_table($code_external_tables,7,'sct_descriptions','ConceptId','Term','Term',array("DescriptionStatus=0","DescriptionType=3"),"");
+define_external_table($code_external_tables,10,'sct2_description','id','term','term',array("active=1"),"");
```

Comment:
Would be interested to see what happens if do the following:
``````
define_external_table($code_external_tables,10,'sct2_description','id','term','term',array("active=1","term LIKE '%(disorder)'"),"");